views/components/GraphTable/index.jsx

```diff
@@ -1,62 +1,89 @@
 import React from 'react'
-import {
-  TableBody,
-  TableCell,
-  TableHead,
-  TableRow,
-  Table,
-  Paper,
-  Toolbar,
-  Button,
-} from '@mui/material'
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import { Paper, Button } from '@mui/material'
+import { SaveAlt } from '@mui/icons-material'
+import Table from '../../tables/Table'
+import { SORT_DIRECTION } from '../../../constants'
 
-const GraphTable = ({ graph, onGetCsv }) => {
+import './GraphTable.css'
+
+const GraphTable = ({
+  graph,
+  onGetCsv,
+  page,
+  rowsPerPage,
+  handleChangePage,
+  handleChangeRowsPerPage,
+  comparator,
+  stableSort,
+  sortBy,
+  sortDirection,
+  onSort,
+}) => {
   if (!graph) return null
 
   const { tableColumns, tableRows } = graph.graphTable
+  const headers = [
+    {
+      dataProperty: 'site',
+      label: 'Network',
+      sortable: true,
+    },
```

Comment:
it looked like this was already being added in the bar chart table service?

Reply:
It used to be like that, now this and the other headers are created on this graph page dynamically, with the properties we require for our table.

Reply:
so can we move this there too?

Reply:
I want to make sure I understand what's being asked..

```javascript
    {
      dataProperty: 'site',
      label: 'Network',
      sortable: true,
    },
...tableColumns.map(({ name }) => {
      return {
        dataProperty: name,
        label: name,
        sortable: false,
      }
    }),
```

The suggestion is to have this logic moved to the server..? If that's the question, I'd like to keep this here, for consistency, all of our other tables have their headers defined on their table page, like why should these be any different..?

Reply:
I may be looking at the wrong part of the code with my comment above. It seems like we're adding an extra column on the server side and the matching additional column to each row on the front-end. My suggestion is to do it all in one place, on the server side.

Reply:
Yeah, doing it there made also helped with the csv code, thanks @mikestone14 

[e61d2c6](https://github.com/AMP-SCZ/dpdash/pull/695/commits/e61d2c603020119c39c8112e6167441b83a0a61a)